micrometer-core/src/test/java/io/micrometer/core/instrument/binder/logging/Log4j2MetricsTest.java

```diff
@@ -186,6 +187,27 @@ void asyncLogShouldNotBeDuplicated() throws IOException {
             .until(() -> registry.get("log4j2.events").tags("level", "info").counter().count() == 3);
     }
 
+    // see https://github.com/micrometer-metrics/micrometer/pull/872
```

Comment:
Should we also add `@Issue("#867")`?

Reply:
We can, though I guess 867 is linked to from 872. Obviously out of scope for this PR, but since you reminded me, I wonder if we should get rid of that `@Issue` annotation entirely since we have no tooling that uses it and so it doesn't seem to add value over a comment without an annotation.